DEVELOPER_GUIDE.md

```diff
@@ -162,6 +162,21 @@ extension_hw_greet:
     - "hw-user"
 ```
 
+### Setting up password for demo admin user
+
+This step is a pre-requisite to installing demo configuration. There are two ways password can be set for demo `admin` user:
+1. **Setting via environment variable:** Password be passed by exporting `OPENSEARCH_INITIAL_ADMIN_PASSWORD` variable with a password.
+```shell
+export OPENSEARCH_INITIAL_ADMIN_PASSWORD=<password>
+```
+
+2. **Setting via a text file:** Password can also be supplied by creating a single line text file `opensearch_initial_admin_password.txt` file with `<password>` under `$OPENSEARCH_HOME/config/` folder.
```

Comment:
```suggestion
2. **Setting via a text file:** The password can also be supplied by creating a single line text file `opensearch_initial_admin_password.txt` file with `<password>` under `$OPENSEARCH_HOME/config/` folder.
```

Reply:
@DarshitChanpura Didn't reading password from a file get removed, seems like this should also be updated.

Reply:
Yes, it has been removed